### ISSUE TITLE
Completion is broken when using curly braces in shell scripts

### ISSUE THREAD

**Issue by hermitmaster** [hrsh7th/cmp-path#57]
Completion is broken when using curly braces in shell scripts. `export XDG_CONFIG_HOME="${HOME}/"` for example, offers completions from the root of the filesystem. If I remove the curly braces, it works as expected.

**Comment by ediw8311xht** [comment]
Yeah, this is very annoying and goes against shellcheck standards of using curly braces with variables.

> Prefer putting braces around variable references even when not strictly required.

https://www.shellcheck.net/wiki/SC2250

Working on making a pull request, but I don't know Lua at all, so it may be a bit.
